ni/panels/v1/streamlit_panel_configuration.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package ni.panels.v1;
+
+option cc_enable_arenas = true;
+option csharp_namespace = "NationalInstruments.Panels.V1";
+option go_package = "panelsv1";
+option java_multiple_files = true;
+option java_outer_classname = "StreamlitPanelConfigurationProto";
+option java_package = "com.ni.panels.v1";
+option objc_class_prefix = "NIPS";
+option php_namespace = "NI\\Panels\\V1";
+option ruby_package = "NI::Panels::V1";
+
+// Pack this into PanelService.StartPanelRequest.panel_configuration to start a Streamlit panel.
+message StreamlitPanelConfiguration 
+{
+  // Absolute path of the streamlit script file on disk, or network path to the file. This must end in ".py".
+  string panel_script_path = 1;
```

Comment:
For this and the following field, would it make sense to call this a `URL` instead and require a `file` scheme initially? That is what we did with the [measurement API](https://github.com/ni/ni-apis/blob/main/ni/measurementlink/measurement/v2/measurement_service.proto#L102). That would allow for supporting other storage mechanisms over time without requiring API changes.

Alternatively, our [pin map API](https://github.com/ni/ni-apis/blob/main/ni/measurementlink/pinmap/v1/pin_map_service.proto#L65) requires the file contents to be passed explicitly. I don't know if that would make sense here or not.

Reply:
You could still opt to make this a URL and only accept file urls for the `localhost` initially. It doesn't seem like a stretch to think that we would one day want to support this panel service as the UI for our measurement services. In that case, it seems like it might be convenient to serve up the python script via the server itself rather than the file system. If you start with only a file path, what do you envision the evolution of this API looks like for that case?

Reply:
Changed to `panel_script_url`.